src/bundle/Controller/User/UserEditController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\Controller\User;
+
+use EzSystems\EzPlatformAdminUi\Form\Data\User\UserEditData;
+use EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory;
+use EzSystems\EzPlatformAdminUi\Form\SubmitHandler;
+use EzSystems\EzPlatformAdminUiBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class UserEditController extends Controller
+{
+    /** @var \EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory */
+    private $formFactory;
+
+    /** @var \EzSystems\EzPlatformAdminUi\Form\SubmitHandler */
+    private $submitHandler;
+
+    /**
+     * @param \EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory $formFactory
+     * @param \EzSystems\EzPlatformAdminUi\Form\SubmitHandler $submitHandler
+     */
+    public function __construct(
+        FormFactory $formFactory,
+        SubmitHandler $submitHandler
+    ) {
+        $this->formFactory = $formFactory;
+        $this->submitHandler = $submitHandler;
+    }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function userEditAction(Request $request): Response
```

Comment:
You don't need a form just to provide redirection. For content it made sense because it would create a draft thus it needed some kind of protection from misclicks and CSRF. For users where you don't create drafts, you only display editing form you can use simple redirection. Less code less maintenance hassle.